src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -87,31 +76,32 @@ export default class Database extends Emittery {
       }
       this.directory = directory;
 
-      const store = new Level(directory, LEVEL_OPTIONS) as GanacheLevel;
-      db = store;
+      // specify an empty `prefix` for browser-based leveldown (level-js)
+      const leveldownOpts = { prefix: "" };
+      const store = encode(leveldown(directory, leveldownOpts), levelupOptions);
+      this.#rootStore = store;
+      db = levelup(store);
     }
 
     // don't continue if we closed while we were waiting for the db
     if (this.#closed) return this.#cleanup();
 
     const open = db.open();
-    this.trie = db.sublevel("T", LEVEL_OPTIONS) as GanacheSublevel;
+    const sublevelTrie = sub(db, "T", levelupOptions);
+    this.trie = new TrieDB(sublevelTrie);
```

Comment:
Note for other reviewers: these are the only two runtime lines that have changed in this file when compared against develop.